x-pack/platform/plugins/shared/fleet/server/routes/cloud_connector/handlers.ts

```diff
@@ -19,6 +19,13 @@ export interface CreateCloudConnectorRequest {
   cloudProvider: CloudProvider;
 }
 
+export interface UpdateCloudConnectorRequest {
```

Comment:
nit: this and `CreateCloudConnectorRequest`  above should live in `x-pack/platform/plugins/shared/fleet/common/types/rest_spec`